### PR TITLE
Fix release version tag for release yaml

### PR DESCRIPTION
**Pull request by khrm**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```
```

### CODE REVIEW

File: tekton/publish.yaml [hunk 1]
```diff
@@ -75,6 +75,12 @@ spec:
       sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/controller-service.yaml
       sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/webhook.yaml
       sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/webhook-service.yaml
+      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/300-clustertriggerbinding.yaml
+      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/300-eventlistener.yaml
+      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/300-triggerbinding.yaml
+      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/300-triggertemplate.yaml
```
Comment (dibyom) [inline tekton/publish.yaml:81]: Can we just do a single `sed` command that replaces it in all files e.g. https://github.com/tektoncd/pipeline/blob/master/tekton/publish.yaml#L143?

Comment (khrm) [reply tekton/publish.yaml:81]: Yes, that works. :)

File: tekton/publish.yaml [hunk 2]
```diff
@@ -71,10 +71,7 @@ spec:
       # Change to directory with our .ko.yaml
       cd /workspace/go/src/github.com/tektoncd/triggers
       # Rewrite "devel" to inputs.params.versionTag
-      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/controller.yaml
-      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/controller-service.yaml
-      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/webhook.yaml
-      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/webhook-service.yaml
+      sed -i -e 's/\(pipeline.tekton.dev\/release\): "devel"/\1: "$(inputs.params.versionTag)"/g' -e 's/\(version\): "devel"/\1: "$(inputs.params.versionTag)"/g' /workspace/go/src/github.com/tektoncd/pipeline/config/*.yaml
```
Comment (dibyom) [inline tekton/publish.yaml:74]: should be `triggers.tekton.dev`  and `/workspace/go/src/github.com/tektoncd/triggers/config/*.yaml`  👼

Comment (khrm) [reply tekton/publish.yaml]: Done.

File: tekton/publish.yaml [hunk 3]
```diff
@@ -71,10 +71,7 @@ spec:
       # Change to directory with our .ko.yaml
       cd /workspace/go/src/github.com/tektoncd/triggers
       # Rewrite "devel" to inputs.params.versionTag
-      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/controller.yaml
-      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/controller-service.yaml
-      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/webhook.yaml
-      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/triggers/config/webhook-service.yaml
+      sed -i -e 's/\(pipeline.tekton.dev\/release\): "devel"/\1: "$(inputs.params.versionTag)"/g' -e 's/\(version\): "devel"/\1: "$(inputs.params.versionTag)"/g' /workspace/go/src/github.com/tektoncd/triggers/config/*.yaml
```
Comment (dibyom) [inline tekton/publish.yaml:74]: needs to be `triggers.tekton.dev` I think

**Pull request merged**